### PR TITLE
Quick Fixes

### PR DESCRIPTION
**Pull request by iholl**
I fixed the popup closure button, added pos_agent data to the popup and styled the vector layer by pos_agent (this might need a legend but shows which locations you can buy a license and which you can't)

### CODE REVIEW

File: src/js/vendors-layer.js [hunk 1]
```diff
@@ -4,16 +4,16 @@ import GeoJSON from 'ol/format/GeoJSON'
...
+// const vendorsStyle = new Style({
+//   image: new Circle({
+//     radius: 7,
+//     fill: new Fill({ color: [90, 100, 125, 0.5] }),
+//     stroke: new Stroke({
+//       color: [53, 85, 166, 0.5],
+//       width: 2
+//     })
+//   })
+// })
```
Comment (kissmygritts) [inline src/js/vendors-layer.js]: @iholl delete these lines if they aren't being used.

File: src/js/vendors-layer.js [hunk 2]
```diff
@@ -22,7 +22,31 @@ const vendorsSource = new VectorSource({
...
+          radius: 7,
+          fill: new Fill({ color: [90, 100, 125, 0.5] }),
+          stroke: new Stroke({
+            color: [53, 85, 166, 0.5],
+            width: 2
+          })
+        })
+      })
+    }
+  }
```
Comment (kissmygritts) [inline src/js/vendors-layer.js]: @iholl, maybe good practice to think about how you might encapsulate this into a function? Hint, the function will take `feature` as an argument.

**Review by kissmygritts — CHANGES_REQUESTED** [review]
Good work on the transparency for the close button. I comletely forgot. Great work on styling the different locations by different colors. If only we can figure out the stupid icon issue!

@iholl, check the code for some comments for fixes before accepting the pull request. Let me know if you have questions.

**Comment by iholl** [comment]
Resolved quick fixes, couldn't devise a function for setting the style, also removed test data from the applicaiton

**Review by kissmygritts — APPROVED** [review]
Thanks

**Pull request merged**